searcch_backend/api/resources/artifact_request_list.py

```diff
@@ -0,0 +1,51 @@
+# logic for /artifacts
+
+from searcch_backend.api.app import db
+from searcch_backend.models.model import *
+from searcch_backend.models.schema import *
+from searcch_backend.api.common.stats import StatsResource
+from flask import jsonify, request
+from flask_restful import reqparse, Resource
+from searcch_backend.api.common.auth import (verify_api_key, has_api_key, has_token, verify_token)
+import logging
+from searcch_backend.api.ticket_creation.antapi_client_conf import AUTH
+
+LOG = logging.getLogger(__name__)
+
+
+class ArtifactRequestListAPI(Resource):
+    def __init__(self):
+        # self.reqparse = reqparse.RequestParser()
+        # self.reqparse.add_argument(name='artifact_group_id',
+        #                            type=int,
+        #                            required=True,
+        #                            help='missing artifact_group_id in query string')
+        super(ArtifactRequestListAPI, self).__init__()
+
+    # Return value: requestToTicketIDObj: keys = Artifact Request IDs for a given user; value = Corresponding ticket status for the artifact request
+    def get(self):
+        # args = self.reqparse.parse_args()
+        if has_api_key(request):
+            verify_api_key(request)
+        
+        user_id = None
+        if has_token(request):
+            login_session = verify_token(request)
+            user_id = login_session.user_id
+        list_of_requests_tuples = []
+        if user_id:
+            list_of_requests_tuples = db.session.query(ArtifactRequests).with_entities(ArtifactRequests.artifact_group_id,ArtifactRequests.ticket_id).filter(user_id == ArtifactRequests.requester_user_id).all()
+
+        LOG.error("Paul Kurian!")
```

Comment:
If you want to keep the debugging statements, I'd suggest changing the header to something else meaningful ("requested tuples:") or something.

Reply:
Ah that's embarrassing, I meant to take those out! I don't think logging the requested tuples tuples is required any more so I have taken them out in [ee5ab5c](https://github.com/STEELISI/classnet-backend/pull/20/commits/ee5ab5ce9782e119865a956bcff7b5c84c084aa3)